net/instaweb/rewriter/url_left_trim_filter.cc

```diff
@@ -65,6 +65,7 @@ void UrlLeftTrimFilter::StartElementImpl(HtmlElement* element) {
     resource_tag_scanner::ScanElement(
         element, driver()->options(), &attributes);
     for (int i = 0, n = attributes.size(); i < n; ++i) {
+      if (attributes(i).category == semantic_type::kSrcSetImage) {continue;}
```

Comment:
put the `continue;` on a separate link, which makes it easier to read and debug.

Reply:
Sure